contrib/helm/pganalyze-collector/values.yaml

```diff
@@ -21,6 +21,15 @@ extraEnv: {}
 #   DB_ALL_NAMES: true (monitor all databases on the server)
 #   DB_USERNAME: your_monitoring_user
 
+# -- Environment variables to be passed to the container
+# Config settings can be defined in raw form, for use with externally maintained env value sources (configMapKeyRef, fieldRef, resourceFieldRef, secretKeyRef)
+extraEnvRaw: []
```

Comment:
Just a memo, I can see that this pattern and the name `extraEnvRaw` is used in the other repos too, so this looks good 👍 

(e.g. https://github.com/apache/superset/commit/b35645c3f4db8526c12ef88d9056a348fcc1f249)